vendor/k8s.io/kubernetes/pkg/util/iptables/iptables.go

```diff
@@ -323,6 +326,27 @@ func (runner *runner) RestoreAll(data []byte, flush FlushFlag, counters RestoreC
 	return runner.restoreInternal(args, data, flush, counters)
 }
 
+// Basically duplicate iptables 1.6.x xtables_lock() function.  We don't bother
+// trying to acquire the 1.4.x-style lock socket since both F25 and RHEL7
+// use the lockfile approach.
+func grabIptablesLock() (*os.File, error) {
+	// First acquire the iptables 1.6.x lockfile (eg, Fedora 25+ and RHEL7)
```

Comment:
There's no "second" though... is that comment left over from an earlier version of the patch that had 1.4.x code too?